src/main/resources/openapi/usermanagement.yaml

```diff
@@ -316,7 +316,6 @@ components:
     UserName:
```

Comment:
Does this need to become `UserFullName` to avoid any confusion with the old `user_name` column?

Reply:
We have some FE tickets in progress that would be impacted by changing the attribute name, will consider this as an improvement.